recipe/meta.yaml

```diff
@@ -27,6 +27,15 @@ requirements:
     - crosstool-ng >=1.23.0
 
 outputs:
+  - name: sysroot_{{ target_platform }}
+    version: "{{ gnu }}"
+    build:
+      number: {{ build_num }}
+      track_features:
+        - glibc_{{ gnu | replace(".", "_") }}    # [(target_platform == "linux-64" or target_platform == "linux-32") and ctng_vendor =="cos7"]
+    about:
+      summary: Metapackage for targeting a particular version of sysroot
```

Comment:
This shouldn't be a metapackage. Everything in `gcc_impl_linux-64` under `$PREFIX/$HOST/sysroot` should go in this package instead.

Reply:
Sorry, can you stub out what you mean here?  Aren't those files already being captured by other outputs here?

Reply:
They are currently in `gcc_impl_linux-64`, but they should really be in the new sysroot package.

Reply:
I still don't get it, can you please provide an example?

Reply:
Sorry for the late response. A file like `x86_64-conda_cos6-linux-gnu/sysroot/lib/libc.so.6` belongs in the `sysroot` package, not in `gcc_impl` package.

Reply:
Ok!